docs: add release notes for July 27-31, 2026 (#25084)
newrelic/docs-website: src/content/docs/release-notes/docs-release-notes/docs-7-31-2026.mdx

---
subject: "Docs"
releaseDate: '2026-07-31'
version: 'July 27 - July 31, 2026'
---

## New docs

> * Added [eBPF Logs Public Preview](/docs/ebpf/logs) documentation, including a configuration reference and best practices.

Reviewer: ```suggestion
* [eBPF Logs Public Preview](/docs/ebpf/logs) documentation is now available, including a configuration reference and best practices.
```

> * Added [custom certificate support](/docs/synthetics/synthetic-monitoring/private-locations/job-manager-configuration/#custom-ca-certificates) documentation for configuring custom CA certificates in private locations.

Reviewer: ```suggestion
* [Custom certificate support](/docs/synthetics/synthetic-monitoring/private-locations/job-manager-configuration/#custom-ca-certificates) documentation is now available for configuring custom CA certificates in private locations.
```

### Major changes

> * **Retired Mobile Crash REST API v1** documentation page. The deprecated v1 API has been superseded by v2.

Reviewer: Shouldn't this be part of EOL? also I guess this is the NRQL API version, this needs to be checked again

* Updated [Azure VNet Flow Logs](/docs/network-performance-monitoring/setup-performance-monitoring/cloud-flow-logs/azure-vnet-flow-log-monitoring) monitoring setup with multi-plan support (`FlexConsumption`, `ElasticPremium`, `Basic`, `Consumption`), FedRAMP/US Gov Cloud endpoint, and private-mode outbound routing documentation.

### Minor changes

> * Updated [Usage Plan terms](/docs/licenses/license-information/usage-plans/new-relic-usage-plan) to add "Evaluation Features" definition and temporary free evaluation period clause under Pay As You Go.

Reviewer: How is this a minor update? We have updated the Pay As You Go information

> * Updated [Usage Plan terms](/docs/licenses/license-information/usage-plans/new-relic-usage-plan) to add "Evaluation Features" definition and temporary free evaluation period clause under Pay As You Go.

Reviewer: ```suggestion
* Updated [Usage Plan terms](/docs/licenses/license-information/usage-plans/new-relic-usage-plan) to include an "Evaluation Features" definition and a temporary free evaluation period clause under Pay As You Go.
```

> * Updated [Account Management](/docs/accounts/accounts-billing/account-setup/downgradecancel-account) page with a reference article on viewing relationships between accounts.

Reviewer: ```suggestion
* Updated [Account Management](/docs/accounts/accounts-billing/account-setup/downgradecancel-account) to include a reference article on viewing relationships between accounts.
```

* Updated [Java Agent compatibility requirements](/docs/apm/agents/java-agent/getting-started/compatibility-requirements-java-agent).

> * Updated [OpenTelemetry MSSQL](/docs/opentelemetry/database/mssql/introduction) documentation with new configuration information.

Reviewer: ```suggestion
* Updated [OpenTelemetry MSSQL](/docs/opentelemetry/database/mssql/introduction) documentation with new configuration details.   
```

> * Improved [Change Tracking Events API](/docs/change-tracking/change-tracking-events) attribute reference by splitting into Required and Optional sections for faster onboarding.

Reviewer: ```suggestion
* Updated [Change Tracking Events API](/docs/change-tracking/change-tracking-events) attribute reference by splitting required and optional sections for faster onboarding. 
```

> * Updated [Agent Control uninstall commands](/docs/new-relic-solutions/new-relic-one/install-configure/uninstall-agent) to fix incorrect commands.

Reviewer: ```suggestion
* Fixed [Agent Control uninstall commands](/docs/new-relic-solutions/new-relic-one/install-configure/uninstall-agent) to correct incorrect commands.
```
